core/lib/l1_contract_interface/src/i_executor/methods/commit_batches.rs

```diff
@@ -1,33 +1,44 @@
-use std::sync::Arc;
-
-use zksync_types::{
-    commitment::L1BatchWithMetadata, ethabi::Token,
-    l1_batch_commit_data_generator::L1BatchCommitDataGenerator,
-};
+use zksync_types::{commitment::L1BatchWithMetadata, ethabi::Token};
 
 use crate::{
-    i_executor::structures::{CommitBatchInfo, StoredBatchInfo},
+    i_executor::structures::{CommitBatchInfoRollup, CommitBatchInfoValidium, StoredBatchInfo},
     Tokenizable, Tokenize,
 };
 
 /// Input required to encode `commitBatches` call.
```

Comment:
To add clarity: "...for a contract running in a Rollup mode". And similar additions can be made in other places.

Reply:
Good call.